common/containers/PageNotFound/index.ts

```diff
@@ -0,0 +1,3 @@
+import PageNotFound from './PageNotFound';
+
+export default PageNotFound;
```

Comment:
This is pretty minor, but the `container` nomenclature is for connected components. Since the page not found is just a dumb component, it could live in the components folder.